designsafe/static/scripts/data-depot/components/projects/publication-preview/publication-preview-other.component.js

```diff
@@ -32,7 +33,7 @@ class PublicationPreviewOtherCtrl {
             editTags: false,
         };
 
-        if (!this.data || this.data.listing.path != this.filePath) {
```

Comment:
I think this section has been refactored a bit because there is no file listing displayed for the publication preview for "other". You might notice this when you try to resolve conflicts.